content/02.intro.md

```diff
@@ -1,12 +1,19 @@
 
-Deep learning (DL) is a subfield of machine learning (ML) focusing on artificial neural networks with many layers, which are increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
-In many cases, novel biological insights have been revealed through careful evaluation of DL methods ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z].
-However, for researchers and students entirely new to this area and those experienced in using classical ML methods (_e.g._ linear regression), using DL correctly can be a daunting task.
-The lack of concise recommendations for biological applications of DL poses further a challenge for newcomers wishing to apply state-of-the-art DL in their research.
-As DL is an active and specialized research area, detailed resources are rapidly rendered obsolete, and only few resources articulate general DL best practices to the scientific community broadly and the biological community specifically.
-To address this issue, we solicited input from a community of researchers with varied biological and deep learning interests, who wrote this manuscript collaboratively using the GitHub version control platform [@url:https://github.com/Benjamin-Lee/deep-rules] and Manubot [@url:https://greenelab.github.io/meta-review/].
-
-In the course of our discussions, several themes became clear: the importance of understanding and applying ML fundamentals [@doi:10.1186/s13040-017-0155-3] as a baseline for utilizing DL, the necessity for extensive model comparisons with careful evaluation, and the need for critical thought in interpreting results generated by means of DL, among others.
-Ultimately, the tips we collate range from high-level guidance to the implementation of best practices, and it is our hope that they will provide actionable, DL-specific advice for both new and experienced DL practitioners alike who would like to employ DL in biological research.
-By increasing the accessibility of DL for applications in biological research, we aim to improve the overall quality and reporting of DL in the literature, enabling more researchers to utilize these state-of-the art modeling techniques.
+Machine learning (ML) is a branch of computational science organized around the learning by computers via algorithms of relationships between dataset variables. Traditionally this has referred to statistical methods such as linear and logistic regression, but as the scale and complexity of data have increased, many new ML methods have been developed to model and learn increasingly complex relationships.
+One powerful approach for learning complex (i.e. linear and non-linear) relationships between input and output variables is called the artificial neural network, which makes use of a number of serialized regression frameworks (i.e. layers) in order to learn such complexity (cite).
```

Comment:
usually i.e. should be followed by a comma. "i.e." -> "i.e.,"

Reply:
I don't understand the "serialized regression frameworks" -- maybe "series of regression models" but it's still very confusing to say this. Can we replace

"called the artificial neural network, which makes use of a number of serialized regression frameworks (i.e. layers)" 


with sth along the lines of

"called the artificial neural network, which makes use of a series of affine transformations accompanied by non-linear activation functions(i.e., the neural network layers)"

Reply:
I worry the change @rasbt proposed may be incomprehensible to beginners. Someone who doesn't know what a neural network is unlikely to have it clarified by describing it as a "series of affine transformations accompanied by non-linear activation functions". 

What about:

```suggestion
One powerful approach for learning complex (that is, both linear and non-linear) relationships between input and output variables is called an artificial neural network.
In these networks, which are modeled on the human brain, artificial neurons arranged into layers "fire" based on the output of the previous layer combined with a modifiable parameter, called a weight. 
Thus, the process of "training" a neural network is in fact the tuning of these weights in such a way that a metric, called a cost function, is optimized.
```

Reply:
@Benjamin-Lee Yea, that what I was thinking too - that describing it in the introduction as a "series of affine transformations accompanied by non-linear activation functions" would likely still be somewhat inaccessible for many biologists. I like your suggested edit - I think that may be clearer for a wider audience.